SerialPrograms/Resources/PokemonSV/MenuOptionsOCR.json

```diff
@@ -0,0 +1,297 @@
+		"slow": [ "Lente" ],
+		"average": [ "Moyenne" ],
+		"normal": [ "Normale" ],
+		"fast": [ "Rapide" ],
+		"on": [ "Activer" ],
+		"off": [ "Désactiver" ],
+		"manual": [ "Choix manuel" ],
+		"automatic": [ "Envoi automatique" ],
+		"regular": [ "Mode normal" ],
+		"inverted": [ "Mode inversé" ],
+		"close": [ "Rapprochée" ],
+		"far": [ "Éloignée" ]
+	},
+	"ita": {
+		"text-speed": [ "Velocità testo" ],
+		"send-to-boxes": [ "Invio automatico ai box" ],
+		"autosave": [ "Salvataggio automatico" ],
+		"skip-cutscenes": [ "Salta i filmati" ],
+		"background-music": [ "Musica" ],
+		"sound-effects": [ "Effetti sonori" ],
+		"pokemon-cries": [ "Versi Pokémon" ],
+		"vertical-camera-controls": [ "Telecamera (asse verticale)" ],
+		"horizontal-camera-controls": [ "Telecamera (asse orizzontale)" ],
+		"show-nicknames": [ "Visualizza soprannomi" ],
+		"give-nicknames": [ "Dai soprannomi" ],
+		"skip-move-learning": [ "Salta gestione mosse" ],
+		"controller-rumble": [ "Vibrazione controller" ],
+		"helping-functions": [ "Funzione recupero" ],
+		"camera-support": [ "Telecamera automatica" ],
+		"camera-interpolation": [ "Velocità di interpolazione" ],
+		"camera-distance": [ "Distanza telecamera" ],
+		"controls-while-flying": [ "Impostazione per Volo libero" ],
+		"slow": [ "Lenta" ],
+		"normal": [ "Normale" ],
+		"fast": [ "Veloce" ],
+		"on": [ "Sì" ],
+		"off": [ "No" ],
+		"inverted": [ "Invertita" ],
+		"close": [ "Vicina" ],
+		"far": [ "Lontana" ]
+	},
+	"kor": {
+		"text-speed": [ "이야기의 속도" ],
+		"send-to-boxes": [ "지닌 포켓몬/박스" ],
+		"autosave": [ "자동 리포트" ],
+		"skip-cutscenes": [ "영상 스킵", "영상스킴" ],
+		"background-music": [ "BGM" ],
+		"sound-effects": [ "효과음" ],
+		"pokemon-cries": [ "울음소리" ],
+		"vertical-camera-controls": [ "카메라 위/아래" ],
+		"horizontal-camera-controls": [ "카메라 왼쪽/오른쪽" ],
+		"show-nicknames": [ "닉네임 표시" ],
+		"give-nicknames": [ "닉네임 등록" ],
+		"skip-move-learning": [ "기술 배우기 스킵", "기술배우기스킴" ],
+		"controller-rumble": [ "컨트롤러 진동" ],
+		"helping-functions": [ "도우미 기능" ],
+		"camera-support": [ "카메라 서포트" ],
+		"camera-interpolation": [ "카메라 천천히 보정" ],
+		"camera-distance": [ "카메라 거리" ],
+		"controls-while-flying": [ "라이드 기술: 비행 상/하" ],
+		"slow": [ "느리게" ],
+		"normal": [ "보통" ],
+		"fast": [ "빠르게" ],
+		"on": [ "On" ],
+		"off": [ "Off" ],
+		"manual": [ "스스로 선택한다" ],
+		"automatic": [ "자동으로 보낸다" ],
+		"inverted": [ "반전" ],
+		"close": [ "가까이" ],
+		"far": [ "멀리" ],
+		"show": [ "본다" ],
+		"dont-show": [ "보지 않는다" ]
+	},
+	"jpn": {
+		"text-speed": [ "話のはやさ" ],
+		"send-to-boxes": [ "手持ち／ボックス", "手持ちノホックス" ],
+		"autosave": [ "おまかせレポート" ],
+		"skip-cutscenes": [ "ムービースキップ", "ムーヒヒースキッフ" ],
+		"background-music": [ "BGM" ],
+		"sound-effects": [ "SE" ],
+		"pokemon-cries": [ "鳴き声" ],
+		"vertical-camera-controls": [ "カメラ上ノ下" ],
+		"horizontal-camera-controls": [
+			 "カメラ左／右", 
+			 "カメラ左ノ右", 
+			 "カメラをノ右", 
+			 "カメラをノ向",  
+			 "カメラをノ君", 
+			 "カメラをノ司", 
+			 "カメラ左ノ向", 
+			 "カメラ左ノ司", 
+			 "カメラ左ノ君" 
```

Comment:
What's going on here? Are these all misread candidates?

Reply:
Yes. All slugs with multiple values are for misread candidates. That slug was particularly bad for some reason. I'm guessing it's because the first 3 characters are shared with other slugs (e.g. horizontal-camera-controls would be misread as vertical-camera-controls). Also, the misread would for some reason change between runs, which is why I had all those different permutations.